pkgs/by-name/wo/wordlists/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, nmap
+, rockyou
+, runtimeShell
+, seclists
+, symlinkJoin
+, tree
+, wfuzz
+, lists ? [
+    nmap
+    rockyou
+    seclists
+    wfuzz
+  ]
+}:
+
+symlinkJoin rec {
+  pname = "wordlists";
+  version = "unstable-2023-10-10";
```

Comment:
Is there any reason for this to be unstable if we are the one's maintaining it?

Reply:
I don't know, I just went with the version scheme I know which is either use the upstream version which we don't have here or use unstable-date-of-last-edit 

Feel free to make suggestions on how we should version it.